dotcom-rendering/src/web/components/Nav/ExpandedMenu/Columns.tsx

```diff
@@ -174,28 +247,44 @@ export const Columns: React.FC<{
 			)}
 
 			<li role="none">
-				<ThemeProvider theme={{ ...buttonThemeBrand }}>
-					<div css={searchBar}>
-						<LinkButton
-							href="https://www.google.co.uk/advanced_search?q=site:www.theguardian.com"
-							tabIndex={-1}
-							className="selectableMenuItem"
-							priority="secondary"
-							icon={
-								<SvgMagnifyingGlass
-									isAnnouncedByScreenReader={true}
-									size="medium"
-								/>
-							}
-							aria-label="Search with google"
-							data-link-name="nav2 : search : submit"
-							type="submit"
-						>
-							Search
-						</LinkButton>
-					</div>
-				</ThemeProvider>
+				<form css={searchBar} action="https://www.google.co.uk/search">
+					<TextInput
+						hideLabel={true}
+						label="Search input"
+						cssOverrides={searchInput}
+						name="q"
+						placeholder="Search"
+						data-link-name="nav2 : search"
+						tabIndex={-1}
+					/>
 
+					<Label hideLabel={true} text="google-search">
+						<div css={searchGlass}>
+							<SvgMagnifyingGlass
+								isAnnouncedByScreenReader={true}
+								size="medium"
+							/>
+						</div>
+					</Label>
+					<Button
+						icon={
+							<SvgArrowRightStraight
+								isAnnouncedByScreenReader={false}
+								size="medium"
+							/>
+						}
+						aria-label="Search with google"
+						cssOverrides={searchSubmit}
```

Comment:
Not sure where the css is that needs to change for this but the focus styles for this button seem a bit off (see image)

<img width="146" alt="Screenshot 2022-10-25 at 10 21 58" src="https://user-images.githubusercontent.com/1336821/197736532-e5ab2a9b-db67-40af-bf24-072f21801a22.png">